scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -208,6 +208,15 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                     .EnsureSuccessful();
             }
 
+            // build ProjectsWithTests\MultipleFrameworkProject for net451
```

Comment:
Yup, fixed it in the previous commit. I'm not sure if the build bots are working though.